scheduler-worker/src/test/java/gov/nasa/jpl/aerie/scheduler/worker/services/SchedulingIntegrationTests.java

```diff
@@ -2744,7 +2746,128 @@ export default () => Goal.CoexistenceGoal({
     assertEquals(SerializedValue.of("fromStem"), peelBanana.serializedActivity().getArguments().get("peelDirection"));
   }
 
-    /**
+  @Test
+  void testJustAfter() {
```

Comment:
Minor: Can there be two additional tests added that uses `WindowProperty.END` instead of `WindowProperty.START`? I believe you may even be able to do this by converting the test into a [parametrized test](https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests), rather than creating two new tests.

Reply:
Sounds good.

Reply:
Done